DESCRIPTION

```diff
@@ -23,3 +23,5 @@ License: MIT + file LICENSE
 Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.1.2
+Imports: 
+    katherinemansfieldr
```

Comment:
Thanks Lynn. I guess we don't need the dependencies for this milestone or am I mistaken?

Reply:
You are right Paniz, dependencies are not required for this milestone. I just added it in because it is mentioned in my function documentation. I think eventually we will need this dependency for the count_punc function to work.